packages/edit-navigation/src/components/header/index.js

```diff
@@ -1,48 +1,39 @@
 /**
  * WordPress dependencies
  */
+import { __ } from '@wordpress/i18n';
 import { NavigableToolbar } from '@wordpress/block-editor';
-import { DropdownMenu } from '@wordpress/components';
 import { useViewportMatch } from '@wordpress/compose';
 import { PinnedItems } from '@wordpress/interface';
-import { __, sprintf } from '@wordpress/i18n';
-import { decodeEntities } from '@wordpress/html-entities';
 
 /**
  * Internal dependencies
  */
+import MenuActions from './actions';
```

Comment:
Nit: I wonder whether it would be easier not to re-use the word `actions` as a file/component name given we use this in stores.

Reply:
What do you think about `dropdown.js` and `MenuDropdown`?

P.S. I partially adopted this naming convention from FSE 😄

Reply:
Done ✅